### PR TITLE
Add Hera (resolves #252)

### LINKED ISSUE #252: Add Hera
https://github.com/bioturing/hera

### PR DESCRIPTION
**Pull request by jvivian**

### CODE REVIEW

**Comment by jvivian** [comment]
@fnothaft — Ready for review

File: hera/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+# Install cmake 3.9.1
+WORKDIR /
+# git clone hera since we're here
+RUN git clone https://github.com/bioturing/hera.git
+# On with cmake
+RUN wget https://cmake.org/files/v3.9/cmake-3.9.1.tar.gz
+RUN tar -xzvf cmake-3.9.1.tar.gz
+WORKDIR /cmake-3.9.1
```
Comment (fnothaft) [inline hera/Dockerfile]: We've got multiple images that install cmake, can we factor that out into a separate image we use? I'm fine if you open an issue for later and don't make that change in this PR.

File: hera/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+RUN make install
+
+# install hera from source
+WORKDIR /hera
+RUN git checkout 53dd2a1927d86d2299f16cf265d3f0e259c0e3df
+RUN chmod +x build.sh
+RUN ./build.sh
+
+# Boilerplate
+RUN mkdir /data
```
Comment (fnothaft) [inline hera/Dockerfile]: I don't think you need to make `/data`, when you `-v` mount, it should be created. I would remove this.

**Review by fnothaft — CHANGES_REQUESTED** [review]
1 small change + a suggested follow up. Otherwise LGTM.

File: hera/Dockerfile (hunk 2 above)
Comment (jvivian) [reply hera/Dockerfile]: I thought the convention was that the WORKDIR is set to **/data** so it's the CWD by default.

File: hera/Dockerfile (hunk 1 above)
Comment (jvivian) [reply hera/Dockerfile]: Great idea, I'll make that now and rebase

File: hera/Dockerfile (hunk 2 above)
Comment (fnothaft) [reply hera/Dockerfile]: I don't think we've typically done that.

Comment (fnothaft) [reply hera/Dockerfile]: Oops. You are correct. Disregard me!

**Comment by jvivian** [comment]
@fnothaft — Ready for review

**Review by fnothaft — APPROVED** [review]
LGTM! Thanks @jvivian. I'll squash and merge when the build passes.

**Comment by jvivian** [comment]
@fnothaft — Small warning if using the cmake base image, `mkdir /data` fails b/c it already exists in the base image which is why I changed it to `mkdir -p /data`.  Another thing is that the base image changes the default workdir from `/` to `/data`, which is another quirky thing to remember.

**Comment by fnothaft** [comment]
@jvivian noted! Seems fine to me though; a small cost to pay for factoring the image out.

**Comment by fnothaft** [comment]
Merged! Thanks @jvivian!

**Pull request merged**